crates/tako/src/worker/rpc.rs

```diff
@@ -203,6 +207,11 @@ pub async fn run_worker(
         }
     };
 
+    let overview_update_interval = match overview_update_interval {
+        None => Either::Left(futures::future::pending()),
```

Comment:
How so I make it explicit, why does it break even though the ```hw_polling_process``` part looks exactly the same?